dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/EmbeddedServerImpl.java

```diff
@@ -380,12 +381,32 @@ public Result call() {
      * of trying to stop the server.
      */
     private class StopOperation implements Callable<Result> {
+
+        private boolean isForceShutdown = false;
+
         /**
-         * No arguments yet: someday there may be, we're just being flexible.
+         * Handles argument parsing for stop command.
          *
          * @param arguments
          */
-        StopOperation(String... arguments) {}
+        StopOperation(String... arguments) {
+            List<String> cmdArgs = new ArrayList<String>(Arrays.asList(arguments));
```

Comment:
There's nothing wrong with what you did here, I'm just curious why you created an ArrayList when the `Arrays.asList()` is going to give you back a `List`.  I didn't see you using any specific methods on `ArrayList`.

Reply:
I think I based my arguments parsing off server start which uses an ArrayList. That's a good point though, I didn't realize that I am not using specific ArrayList methods. Going to get rid of the ArrayList cause we don't need it.